<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserDirPermissionDialog</class>
 <widget class="QWidget" name="UserDirPermissionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户目录权限</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>目录</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>权限</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNewDir">
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>新增(&amp;N)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModifyDir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>修改(&amp;M)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonDeleteDir">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>81</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>删除(&amp;D)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
